[Chore] v15.0-rc1 release (#540)
serokell/tezos-packaging: docs/systemd.md

networks.

`tezos-node` packages provide multiple services out of the box:

> `tezos-node-kathmandunet`, and `tezos-node-mainnet` that run
> `kathmandunet` and `mainnet` networks respectively.

Reviewer: This is incomplete, I suggest changing it to:
```suggestion
- `tezos-node-kathmandunet`
- `tezos-node-limanet`
- `tezos-node-ghostnet`
- `tezos-node-mainnet`

which run on the respective networks.
```
